samples/web/content/testrtc/js/camtest.js

```diff
@@ -7,26 +7,6 @@
  */
 'use strict';
 
-// Test spec
```

Comment:
I am fine with removing it. Since at some moment this comment will not match the test anymore.